dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/AlertGroupMapper.xml

```diff
@@ -32,6 +32,15 @@
         </if>
         order by update_time desc
     </select>
+    <select id="queryAlertGroupName" resultType="Map">
```

Comment:
Do you want to add a pagination query? This is not a pagination query, maybe you need to add `limit pageNo*pageSize, pageSIze`(you need to confirm)

Reply:
I use MyBatis-Plus, so there is no need to add.

Reply:
Ok